src/NUnitFramework/framework/Internal/AsyncEnumerableAdapter.NetCore.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+#if !NETFRAMEWORK
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace NUnit.Framework.Internal
+{
+    internal static partial class AsyncEnumerableAdapter
+    {
+        private static partial bool TryGetAsyncBlockingEnumerable(object enumerable, out IEnumerable<object?>? result)
+        {
+            if (enumerable is IAsyncEnumerable<object> asyncEnumerable)
+            {
+                // Allow for lazily enumeration
+                result = new AsyncEnumerableWrapper(asyncEnumerable);
+                return true;
+            }
+
+            result = default;
+            return default;
```

Comment:
```suggestion
            return false;
```

It might be the same value, but reads more logical.